next-steps.md

```diff
@@ -0,0 +1,74 @@
+# Next Steps after `azd init`
```

Comment:
this talks about "after `azd init`", however there is no instruction for that anywhere. could this document include that step as well?

Reply:
This file gets generated after one run `azd init`. Shouldn't be in here. Missed it when I cleaned the repo. Will remove.